docs/rules/alphabetical-paths.md

```diff
@@ -0,0 +1,102 @@
+#  Ensure that required paths are in alphabetical order (alphabetical-paths)
+
+## Rule Details
+
+This rule checks whether the required paths in `define` and `require` statements are order alphabetically or not.
+
+### Options
+
+This rule takes two arguments. First argument is optionoal and can be "fullpath", or "basename". If it is "fullpath" then whole path strings are checked for right order. If "basename", then just basename parts of required paths are checked. The default is "fullpath".
+
+The second argument is also optional and will be used only when the first argument is manually set to "fullpath". It can be "always", or "never". If it is "always" then it checks at first just dirnames of paths and then, if they are equal, it checks basenames. If it is "never" then it just checks entire paths. The default is "always".
```

Comment:
"always"/"never" are unclear names here, because it's not clear what is always or never being done/considered.

Given that the second argument is used only when the first argument is fullpath, I think you would do better with one argument with 3 possible values. Maybe "basename", "fullpath", or "basename-then-fullpath", assuming I'm understanding the implementation correctly?